src/components/ClassifierList/ClassifierList.tsx

```diff
@@ -31,11 +44,11 @@ export const ClassifierList = () => {
 
       <Collapse in={collapsed} timeout="auto" unmountOnExit>
         <List component="div" dense disablePadding>
-          <PredictClassifierListItem />
-
           <FitClassifierListItem />
 
-          <EvaluateClassifierListItem />
+          <PredictClassifierListItem disabled={noFittedModel} />
```

Comment:
I might be wrong but here (and on line 51), couldn't we just have written `disabled={!fitted}` -- and then we wouldn't even need the `useEffect` above and the local state?

Reply:
I changed that in a local commit. I also disabled predicting/evaluating while a model is currently training.